cmd/server/setup.go

```diff
@@ -339,6 +339,11 @@ func setupMetrics(g *errgroup.Group, _store store.Store) {
 		Name:      "repo_count",
 		Help:      "Total number of repos.",
 	})
+	agents := promauto.NewGauge(prometheus.GaugeOpts{
+		Namespace: "woodpecker",
+		Name:      "agents_available_count",
```

Comment:
Naming is confusing and form the look can't be understood if that's total agent count or free agent count. If it's first I would name it `agent_count` to be more consistent with name above `repo_count`. For free agent count I would name it `free_agent_count`